CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+INCLUDE(CMakeForceCompiler)
+
+cmake_minimum_required (VERSION 2.8)
+SET(CMAKE_SYSTEM_NAME Generic)
+
+project(mbedTrace)
+
+
+include_directories(${CMAKE_CURRENT_SOURCE_DIR}/include/)
```

Comment:
Is this correct? There's no `include` directory here, the header is in `mbed-trace`.